modules/javafx.graphics/src/test/java/test/javafx/scene/transform/AffineOperationsTest.java

```diff
@@ -2506,13 +2767,18 @@ public void testSetElement() {
                 affine.getTz() + 1000), a);
     }
 
-    @Test(expected=NullPointerException.class)
-    public void testSetElementNullType() {
-        Affine a = affine.clone();
-        a.setElement(null, 0, 0, 0);
+    @ParameterizedTest
+    @MethodSource("getParams")
+    public void testSetElementNullType(Affine affine) {
+        assertThrows(NullPointerException.class, () -> {
+            Affine a = affine.clone();
+            a.setElement(null, 0, 0, 0);
+        });
     }
 
-    @Test public void nonInvertibleExceptionShoudCancelAtomicOperation() {
+    @ParameterizedTest
+    @MethodSource("getParams")
+    public void nonInvertibleExceptionShoudCancelAtomicOperation(Affine affine) {
```

Comment:
this is one humongous test!!!

Reply:
It definitely would be on the podium, especially for the parameter expansion... 181 tests times 32 arguments (5792 tests total).

But the undisputed champion is `TransformOperationsTest` with 81 tests times 116 arguments (9396 tests) =)